app/views/tag/show.html.erb

```diff
@@ -164,8 +164,9 @@
   <div class="box">
     <div class="row">
       <div class="col-md-12">
-
-        <%= render partial: 'tag/show/contributors' %>
+        <% unless @node_type == 'comments' %>
+          <%= render partial: 'tag/show/contributors' %>
+        <% end %>
```

Comment:
Also, could we add a line similar to this for `comments`? It would be added right after this one. Thanks @billymoroney1!

https://github.com/publiclab/plots2/blob/9198f100b36f4d8bcbedf20b79112868e8ce1278/app/views/tag/show/_nav_tabs.html.erb#L91-L95